lib/dal/tests/integration_test/schema/variant/authoring/update_variant.rs

```diff
@@ -125,3 +135,700 @@ async fn update_variant(ctx: &mut DalContext) {
     assert!(updated_default_schema_variant.is_some());
     assert_eq!(updated_default_schema_variant, Some(second_updated_sv_id));
 }
+
+#[test]
+async fn update_variant_with_new_prototypes_for_new_func(ctx: &mut DalContext) {
```

Comment:
Covers a bug that was recently fixed, but added as a regression test
cc @vbustamante @fnichol, this is one that we talked about